plugins/pagerduty/src/components/ChangeEvents/ChangeEventListItem.tsx

```diff
@@ -60,8 +59,9 @@ export const ChangeEventListItem = ({ changeEvent }: Props) => {
     externalLinkElem = (
       <Tooltip title={text} placement="top">
         <IconButton
-          component={Link}
-          to={changeEvent.links[0].href}
+          href={changeEvent.links[0].href}
```

Comment:
Is this switch from using the `Link` component needed? We'd prefer to keep using it so that we get tracking and automatic opening in new tabs where neded.

Reply:
Thanks for taking a look @Rugvip!

It seems that `Link` component is what was causing the error in https://github.com/backstage/backstage/issues/8409

That error seems to do with the react router so I'm thinking the `Link` component was somehow triggering the router leading to the error.

Thats ultimately why I removed the `Link` component and went with the href instead, which solved the issue. If there is a way to use the `Link` component without triggering the router that you know of, that might work. Other than that I don't have any other ideas how to fix the original error.

Reply:
I think the check for whether `changeEvent.html_url` is `null` or not is the thing that fixes it? Are you able to reproduce the issue if you go back to using `Link` but keep that check?

Reply:
@Rugvip thanks for the tip! You were correct. I've added the `Link` components back in.